src/main/java/com/iota/iri/service/milestone/impl/MilestoneSolidifierImpl.java

```diff
@@ -30,12 +30,12 @@
     /**
      * Defines the amount of milestones that we "simultaneously" try to solidify in one pass.
      */
-    private static final int SOLIDIFICATION_QUEUE_SIZE = 10;
+    private static final int SOLIDIFICATION_QUEUE_SIZE = 20;
```

Comment:
Question:
How did you come to the conclusion that 20 is better than 10?

Not that I think that 10 is such a good number... I am actually not sure what this value should be so any answer here will be good

Reply:
I figured it can solidify more milestones per solidifier step, so I simply doubled it.

Reply:
Sounds reasonable :-)